demo_modules/wind/wind.js

```diff
@@ -29,8 +29,7 @@ const ROTATEY = -400;
 
 function loadJson(file) {
   return new Promise((resolve, reject) => {
-    const asset = require('client/asset/asset');
-    d3.json(asset(file))
+    d3.json('/asset/' + file)
```

Comment:
does this make anything more fragile?  What about changes to the url structure or local mode?

Reply:
Prior to this change, it only worked in local mode. On the wall, it requires the assets be distributed to the clients. we actually want to load these from the server since they are updated every 4 hours.  I'm just checking in a change I had to make in order to test it on the wall before the holidays.

Reply:
We should add methods to asset that are named after what they do like: "always read from server" or "pick based on a config"